### ISSUE TITLE
List system requirements on skytube-app.com and GitHub

### ISSUE THREAD

**Issue by Gitoffthelawn** [ram-on/SkyTube#638]
I recommended SkyTube to a friend, and they are receiving an error when trying to get it from F-Droid.  They are using Android 6.0.

I went to skytube-app.com and GitHub to see if Android 6.0 is supported, but I don't see where system requirements are listed on either.

Can you add system requirements to both places?

(And until then, is Android 6.0 supported?)

**Comment by gzsombor** [comment]
Currently, the app is compiled with minSdkVersion 14, so it should work on Android 4.0 - as most of the new versions of the app dependencies require newer Android version, maybe we need raise the requirement to 19 - Android 4.4, or even to Android 5.0.

Unfortunately, I don't have access to the webapp, only @ram-on has. Could you please open a PR about the readme changes?
 What kind of error do they get from F-Droid?

**Comment by Gitoffthelawn** [comment]
@gzsombor Thanks very much for your reply and the details.

1. I have mixed emotions on minSdkVersion being increased.  Obviously, it will prevent older devices from using ST.  But, on the other hand, sometimes that's needed to make progress.  Ideally, there would be 2 versions: one for minSdkVersion 14, and one for higher... but that's more work and maintenance.

2. You mentioned "webapp"... did you mean "website"?  It's just a static page, right?  Or am I not seeing something?

3. You requested I open a PR for readme changes.  I'm not seeing anywhere in the readme that lists requirements, so I'm not seeing anything that requires changes.  Are you simply suggesting that the info be added to the readme?

4. The F-Droid error turned out to be an F-Droid issue or a device issue.  F-Droid thought ST was already installed (along with a bunch of other apps), and so F-Droid couldn't install ST (even though it wasn't installed).  Clearing F-Droid's cache and data fixed the issue.

Thank you again! :)

**Comment by gzsombor** [comment]
1. Yes, I have mixed feeling too - unfortunately it's basically impossible to know, what Android versions are used by SkyTube users. Testing the app on multiple Android version is a big effort, and honestly, I'm more focused to versions, for which I do have hardware. As @atomjack found out in https://github.com/ram-on/SkyTube/issues/620#issuecomment-577304607, it's possible, that no-one tried to use the app, recently, with Android 4 - or no one, who cared to report that is broken. 
2. Sorry, yes, I meant 'website', when I wrote 'webapp' :smiley: 
3. Yes, I thought, maybe adding this information to the README file, could be an easy improvement.

**Comment by Gitoffthelawn** [comment]
1. Yes, it's hard to know.  I have a couple older Android 4.x (4.4 IIRC) devices that I was considering turning into dedicated SkyTube devices.  At the same time, it's a balancing act trying to decide what to support.

It's funny that you mention people reporting/not reporting bugs.  I just tested a *very* popular application, and within 20 minutes I found (and reported) 4 significant bugs.  I couldn't believe no one had reported them.  I figure when an app is free the least I can do is contribute by reporting bugs that I find.

2. :smile:

3. Sounds good.  I'll do it right now.
